vars/githubWorkflowRun.txt

```diff
@@ -0,0 +1,113 @@
+Run workflow on github actions
+
+### Run as step:
+
+```
+  Map runInfo = githubWorkflowRun(repo: "owner/repository", workflow: "build.yml", ref: "main",
+    parameters: [path: "filebeat"], credentialsId: "github-workflow-token")
+```
+
+#### Run asyncronous:
+
+```
+  script {
+    Map args = [
+       repo: "owner/repository",
+       workflow: "build.yml", 
+       ref: "main",
+       parameters: [path: "filebeat"],
+       credentialsId: "github-workflow-token"]
+    int runId = githubWorkflowRun.triggerGithubActionsWorkflow(args)
+    Map runInfo = githubWorkflowRun.getWorkflowRun(args + [runId: runId]) 
+  }
+
+```
+
+### Arguments:
+
+* repo: repositiry owner and name
```

Comment:
We might not require this argument, since `gh` uses a couple of environment variables that are generated on the fly when using the Multibranch Pipelines. See https://github.com/elastic/apm-pipeline-library/blob/79a1ded5d3c66e760483d9cc4b97300de8958fc2/vars/gh.groovy#L42-L50

```suggestion
* repo: repository owner and name
```

Reply:
fixed